examples/async-universal/common/routes.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Route from 'react-router/lib/Route'
+
+import Reddit from './containers/Reddit'
+
+export default(
+  <Route path="/" component={Reddit}>
+    <Route path=":id" component={Reddit}/>
```

Comment:
OK, I get it now. You used to have `Layout` but I asked to remove it, and this is why it looks strange now. As I also write below, please add `Layout` back but make it more useful. You can call it `App` instead and add a header to it that links to the source, like in `real-world` example. I believe this should be enough to use what seems to be a more correct route config (with `IndexRoute` and a `Route` inside a generic app-wide route).